sway-fmt-v2/src/utils/punctuation.rs

```diff
@@ -0,0 +1,30 @@
+use crate::config::items::ItemBraceStyle;
+use crate::Formatter;
+
+/// Handles bracket open scenerio. Checks the config for the placement of the bracket.
+/// Modifies the current shape of the formatter.
+pub fn handle_open_bracket(push_to: &mut String, formatter: &mut Formatter) {
```

Comment:
One thing we need to be careful of, is that if we are using this to maintain brackets for all items that we add a match statement so that it will target each specific case. For now you could add the match statement in and have everything else as `todo!()`

Reply:
An instance where this might occur is in `use` statements where we don't want a space before `{` like `use a:: {b, c, d};` which would break the code.

Reply:
If it's not too much trouble, could you push this to a separate PR as well?

Reply:
👍 opened-up #1968.